Library/Formula/libdistance.rb

```diff
@@ -0,0 +1,12 @@
+class Libdistance < Formula
+  homepage "http://monkey.org/~jose/software/libdistance/"
+  url "http://monkey.org/~jose/software/libdistance/libdistance-0.2.2.tar.gz"
+  sha1 "d516e5f24ae55bcbbf93a4d14c6a53914e5b3a0a"
+
+  def install
+    # The build system is incredibly simple.
+    system "make"
+    include.install "distance.h"
```

Comment:
Not in the GNU makefile, nope.

The BSD makefile does have an `install` target via `bsd.lib.mk`, but that Makefile builds tcl bindings via swig, which is overkill when all I want is the simple C library.

Alternatively, can I create a --with-swig option and then choose between the build systems appropriately? (sorry for an elementary question, this is my first Homebrew formula)